app/javascript/packages/document-capture/components/in-person-location-post-office-search-step.spec.tsx

```diff
@@ -105,7 +105,7 @@ describe('InPersonLocationStep', () => {
         await findByText('in_person_proofing.body.location.po_search.search_button'),
       );
 
-      const error = await findByText('idv.failure.exceptions.internal_error');
+      const error = await findByText('idv.failure.exceptions.post_office_search_error');
```

Comment:
This looks pretty good to me. I think these are the two tests we want to run, I'm just wondering if there are other failures we might encounter? I didn't see anything in the spec for `spec/services/usps_in_person_proofing/proofer_spec.rb` that would seem to indicate other failure states we might encounter, but I wanted to double check.